### ISSUE TITLE
[PM] Create list of staging users we need for logged-in homepage QA

### ISSUE THREAD

**Issue by Samara-Strauss** [department-of-veterans-affairs/va.gov-team#8385]
## Backlog

As soon as we know the V1 use cases we will cover on the logged-in homepage, we should start figuring out which users we can use for QA and testing.

## Tasks

- [x] Create list of all use cases that will be covered on the logged-in homepage
- [ ] Figure out what staging users can be used to test these use cases

## Acceptance criteria

- [ ] All use cases have been outlined
- [ ] All use cases have a staging user

**Note**: We will create separate tickets as needed to get additional staging users set up

**Comment by mshea0606** [comment]
[LIH staging users and use cases](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/logged-in-homepage/2.0-redesign/product/LIH%20staging%20users%20-%20test%20cases.md)

**Comment by mshea0606** [comment]
Currently waiting on staging users from Luke M and Lenae S. We should have those soon.

**Comment by Samara-Strauss** [comment]
Hey Matt, I am reopening this. Thank you for creating the preliminary list, but it's technically not complete since there are some incorrect use cases and since not all of the use cases have staging user info (and many of them do have corresponding staging users).

Can you please do the following?

- [ ] Review the [use case mural board](https://app.mural.co/t/vsa0499/m/vsa0499/1597156877591/25248822b34df6bbf8041a9b11b3150974e51efb) and [data spreadsheet](https://docs.google.com/spreadsheets/d/1fO3VxUj8U2M268GnpmNmlL8Jdl8E1as33SzuwoTneV8/edit#gid=1279308525) that Tressa and Cassandra put together to make sure you understand all of the use cases.
  - Note: Because of the pivot from LIH --> My VA, you do *not* need to have any use cases around education, records, or showing users additional benefits they can apply for. This info will not be moving to My VA from the homepage.
- [ ] Put together a list of all use cases *without* the corresponding staging users. No need to delete the current doc or anything - this is just easier for me to review if it's in a bulleted list by category (eg. LOA1, health care, etc).
- [ ] Have me review that list.

Once we're sure that list is complete, then you can go back, update the current doc, and fill in staging users.

Also, you should know that the `judy.morrison@id.me` staging user does **not** have the same password as our other staging users. This is because it is managed outside of our team. I believe the password is in Slack, but please **do not** post the password in this document. This is a public repo, and we do not want other people getting access to that information. Please check if that is documented in the va.gov-team/sensitive repo, or link to a Slack post or something with the info. But don't put the password here.

Finally, since this was not actually completed in Sprint 31, please do not included it as a completed ticket in the final sprint report.

**Comment by mshea0606** [comment]
leaving this open, will revisit once we have more information needed for staging user requirements

**Comment by Samara-Strauss** [comment]
@mshea0606 is this a duplicate of #18042? Can you reconcile the two? I want to make sure I know what we should be focusing on re: sprint goals for sprint 39.
